.github/workflows/docker.yml

```diff
@@ -0,0 +1,51 @@
+# NOTE: This name appears in GitHub's Checks API and in workflow's status badge.
+name: docker
+
+# Trigger the workflow when:
+on:
+  # A push occurs to one of the matched branches.
+  push:
+    branches:
+      - master
+      - stable/*
+    paths:
+      - docker/**
```

Comment:
I think it would be useful to also re-build the Docker images daily (even when there are no new commits) so we get the latest security fixes from the updated base Docker images, e.g. `ubuntu:18.04`, and early detection when some part of the Docker image process starts to fail.

Something like:
https://github.com/oasisprotocol/oasis-core/blob/d0a8e24f1a022347067012f06677800e7124fe59/.github/workflows/ci-reproducibility.yml#L11-L13

Note that [`on.schedule` events](https://help.github.com/en/actions/reference/events-that-trigger-workflows#scheduled-events-schedule) have the `GITHUB_REF` set to repository's default branch so this will only work for rebuilding the `master` images.

Reply:
This is actually necessary to do since we use path filtering (on `docker/**`) when building the Docker images for the `master` or `stable/*` branches and those images will thus be only very rarely rebuilt.

Reply:
Added.